apps/namadillo/src/App/Swap/SwapSuccess.tsx

```diff
@@ -0,0 +1,168 @@
+import { Heading, Stack } from "@namada/components";
+import anime from "animejs";
+import { useSetAtom } from "jotai";
+import { useCallback, useEffect, useRef } from "react";
+import { CgCheck } from "react-icons/cg";
+import { SwapStatus } from "./state";
+import { swapStateAtom, swapStatusAtom } from "./state/atoms";
+
+export const SwapSuccess = (): JSX.Element => {
```

Comment:
<img width="1479" height="750" alt="Screenshot 2025-11-07 at 7 17 21 PM" src="https://github.com/user-attachments/assets/9a71404c-9cfc-4bae-9713-128706baa93a" />

After a successful swap I get the checkmark and then it goes back here. Are we not showing a Transaction Receipt for the swaps?

Reply:
No we are not, here is the design https://www.figma.com/design/NFyHbLZXBSl3aUsMxtffvV/Namada-End-User-Interface-V1.0---Phase-1--Block-Party?node-id=10113-2166&t=XcYgWVYTci6BBFNL-0. I assume it's because we do not consider swap as a kind of transfer